### PR TITLE
feat: [IDP-123]: custom component exported for IDP

### PR DESCRIPTION
**Pull request by bhavya-sinha**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [x] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [x] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by bhavya-sinha** [comment]
retrigger titlecheck

**Comment by bhavya-sinha** [comment]
retrigger jest

**Comment by bhavya-sinha** [comment]
retrigger jest

**Comment by bhavya-sinha** [comment]
retrigger jest

**Review by OrkoHunter — DISMISSED** [review]

File: src/microfrontends/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@harness/microfrontends",
-  "version": "1.29.0",
+  "version": "1.30",
```
Comment (udayvunnam) [inline src/microfrontends/package.json:3]: 1.30.0 SEMVER, please

**Review by udayvunnam — APPROVED** [review]

**Review by jenil-harness — APPROVED** [review]

**Comment by bhavya-sinha** [comment]
retrigger cypress-pipeline

**Comment by bhavya-sinha** [comment]
retrigger cypress-cv

**Review by kapilkumawat86 — APPROVED** [review]

**Pull request merged**